build/distributions/CentOS/7/x86_64/DVD/ks/dvd/auto.cfg

```diff
@@ -255,10 +287,12 @@ gpgcheck=0
 EOF
 
 if [ $ostype == "CentOS" ]; then
```

Comment:
In case people don't want to run {{simp config}} was the idea. We can remove it at this point.